server/src/dao/connection.ts

```diff
@@ -1,7 +1,8 @@
 import { connect } from 'mongoose'
 import globals from '../config/globals'
+import { logger } from '../util/logger'
 
 export const getConnection = async () => {
   await connect(globals.MONGO_URI)
-  console.log('Base de datos conectada')
+  logger.info('Base de datos conectada')
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/c0dejohn/e-commerce-backend/pullRequest?prid=8292833)
```suggestion
  logger.info("Base de datos conectada")
```